velox/expression/VectorUdfTypeSystem.h

```diff
@@ -951,6 +956,83 @@ struct VectorWriter<Row<T...>> {
   size_t offset_ = 0;
 };
 
+template <typename... T>
+struct VectorWriter<RowWriterT<T...>> {
+  using vector_t = typename TypeToFlatVector<Row<T...>>::type;
+  using exec_out_t = typename VectorExec::resolver<RowWriterT<T...>>::out_type;
+
+  VectorWriter() {}
+
+  void init(vector_t& vector) {
+    rowVector_ = &vector;
+    // Ensure that children vectors are at least of same size as top row vector.
+    initVectorWritersInternal<0, T...>();
```

Comment:
nit (super minor): leave out "Internal" just call it initVectorWriters, same for resizeVectorWritersInternal if you make it private

Reply:
yes sure, here i just copied the old vector writer and customized it.